### PR TITLE
Docs for the new blank_missings option

### PR DESCRIPTION
**Pull request by jjdelc**

### CODE REVIEW

File: source/includes/_endpoint-dataset.md [hunk 1]
```diff
@@ -852,12 +852,13 @@ The following rules apply for all formats:
 
 Some format-specific properties and options:
 
-Format    | Attribute           | Description                                                           | Example
---------- | --------------------| --------------------------------------------------------------------- | --------------------------
-csv       | use_category_ids    | instead of category names export the fields as their numeric ids      | {"use_category_ids": true}
-spss      | var_label_field     | Use the variable's name/description as SPSS variable label            | {"var_label_field": "name"}
-spss      | prefix_subvariables | When True, will only export subvariables and not group them as arrays | {"prefix_subvariables": true}
-all       | include_personal    | Will include the user's personal variables on the exported file       | {"include_personal": true}
+Format    | Attribute           | Description                                                                         | Example
```
Comment (nealrichardson) [inline source/includes/_endpoint-dataset.md]: Much more useful than "Example" would be "Default".

File: source/includes/_endpoint-dataset.md [hunk 2]
```diff
@@ -852,12 +852,13 @@ The following rules apply for all formats:
...
+--------- | --------------------| ----------------------------------------------------------------------------------- | --------------------------
+csv       | use_category_ids    | Instead of category names export the fields as their numeric ids                    | {"use_category_ids": true}
+csv       | blank_missings      | When True, exports missing values as empty string, else will use the missing reason | {"blank_missings": true}
+spss      | var_label_field     | Use the variable's name/description as SPSS variable label                          | {"var_label_field": "name"}
+spss      | prefix_subvariables | When True, will only export subvariables and not group them as arrays               | {"prefix_subvariables": true}
```
Comment (nealrichardson) [inline source/includes/_endpoint-dataset.md]: The "not group them as arrays" is inaccurate, right?

Comment (jjdelc) [reply source/includes/_endpoint-dataset.md]: True, we changed it to only prefix the names

File: source/includes/_endpoint-dataset.md [hunk 3]
```diff
@@ -777,12 +777,13 @@ The following rules apply for all formats:
...
-csv       | use_category_ids    | instead of category names export the fields as their numeric ids      | {"use_category_ids": true}
-spss      | var_label_field     | Use the variable's name/description as SPSS variable label            | {"var_label_field": "name"}
-spss      | prefix_subvariables | When True, will only export subvariables and not group them as arrays | {"prefix_subvariables": true}
-all       | include_personal    | Will include the user's personal variables on the exported file       | {"include_personal": true}
+Format    | Attribute           | Description                                                                         | Default
+--------- | --------------------| ----------------------------------------------------------------------------------- | --------------------------
+csv       | use_category_ids    | Instead of category names export the fields as their numeric ids                    | {"use_category_ids": false}
+csv       | missing_values      | Which value to use for printing missing values; allowed "dot", "reason", "blank"    | {"missing_values": "reason}
```
Comment (nealrichardson) [inline source/includes/_endpoint-dataset.md]: missing a close "

Comment (nealrichardson) [reply source/includes/_endpoint-dataset.md]: And why not just give the value instead of `{key: value}` for the defaults? Seems redundant.

Comment (nealrichardson) [reply source/includes/_endpoint-dataset.md]: Why not just use "" for "blank" and "." for "dot"? That way you can take any user specified string (that isn't "reason") and use that.

Comment (jjdelc) [reply source/includes/_endpoint-dataset.md]: "" is too falsey, I prefer the more explicit option. Cannot be confused with "forgot value" case.

Comment (nealrichardson) [reply source/includes/_endpoint-dataset.md]: I disagree that "" is too falsey, particularly when this field has no boolean meaning. And I don't know what "forgot value" means--if I truly forgot to specify a value, this wouldn't be here at all. 

I naturally have R in mind when I suggest the direct string input. See the 'na' argument to https://stat.ethz.ch/R-manual/R-devel/library/utils/html/write.table.html, which is what writes CSVs in R.

File: source/includes/_endpoint-dataset.md [hunk 4]
```diff
@@ -798,8 +799,14 @@ To pick which variable field to use on the `label` field on the SPSS variables,
...
-Categorical variable values will be exported as the category name by default.
-To use the category ids as data values, include `"use_category_ids": true` in the `"options"` attribute of the POST body.
+By default, categorical variable values will be exported using the category name 
+and missing values will use their corresponding reason string for all variables.
+
+The missing values will be exported with their configured missing reason in
+the CSV file. If specified on the `missing_values` export option, then all
+missing values on all columns will use such string instead of the reason.
+
+Refer to the options described on the table above for the `csv` format to change this behavior.
```
Comment (percious) [inline source/includes/_endpoint-dataset.md:810]: can you describe "blank" and "dot" here?

**Review by percious — CHANGES_REQUESTED** [review]

File: source/includes/_endpoint-dataset.md (hunk 4 above)
Comment (nealrichardson) [reply source/includes/_endpoint-dataset.md:810]: "blank" and "dot" don't exist anymore.

Comment (jjdelc) [reply source/includes/_endpoint-dataset.md:810]: Is this from an outdated version? The current content of the text describes, whatever string you send will be used.

**Pull request merged**